exporter/awsemfexporter/util_test.go

```diff
@@ -20,13 +20,12 @@ import (
 	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
 	agentmetricspb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/metrics/v1"
 	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
+	internaldata "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus"
```

Comment:
This is not following the import grouping rules used by this repo and will fail the lint checks.  Revert the changes to this file.

Reply:
Done too !! Thanks for your insight !!!